src/tides.f90

```diff
@@ -573,4 +547,358 @@ subroutine setTidesInitFlag(this)
 
   end subroutine setTidesInitFlag
 
+!=========================================================================
+!
+!> @anchor      getDelaunay_arg
+!!
+!> @brief       Compute the Delaunay arguments (IERS chapter 5)
+!> @author      Andrea Turchi (ATU)
+!!
+!> @date        <ul>
+!!                <li> 17.08.22 (initial design) </li>
+!!              </ul>
+!!
+!!-----------------------------------------------------------------------
+  subroutine getDelaunay_arg(this, time_mjd, F_vect)
+
+    implicit none
+    class(Tides_class)                  :: this
+    real(dp), intent(in)                :: time_mjd                ! time in modified julian date
+    real(dp), parameter                 :: jd2000 = 2451545.0      ! factor to count julian days in jd2000 time reference
+    real(dp), parameter                 :: conv = 360.0/1296000.0  ! conversion factor from arcseconds to degrees 
```

Comment:
Can you look into libslam, whether we have this conversion there already? If not can you add it at an appropriate spot, possibly slam_astro.f90. libslam needs a new branch then as well.